### PR TITLE
Release 4.5.0rc4

### PR DESCRIPTION
**Pull request by G-D-Petrov**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by G-D-Petrov** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ build:
   # TODO: the specific on python does not work for now, we need to investigate why.
   # skip: true  # [libevent == "2.1.10" and py>39]
   # skip: true  # [libevent == "2.1.10" and py<39]
-  number: 3
+  number: 4
```
Comment (jjerphan) [inline recipe/meta.yaml:30]: Ideally, the build number should be reset to 0 (rc3's should should have been reset to 0, I missed that).
```suggestion
  number: 0
```

**Review by jjerphan — APPROVED** [review]
LGTM given :green_circle: CI checks.

**Comment by G-D-Petrov** [comment]
@conda-forge-admin, please rerender

**Pull request merged**